core/network/rcmgr.go

```diff
@@ -102,7 +102,7 @@ type ResourceManager interface {
 // ResourceScopeViewer is a mixin interface providing view methods for accessing top level
 // scopes.
 type ResourceScopeViewer interface {
-	// ViewSystem views the system wide resource scope.
+	// ViewSystem views the system-wide resource scope.
```

Comment:
This is not correct.

Reply:
Apparently it is either `systemwide` or `system-wide` https://en.wiktionary.org/wiki/system-wide#English so `system-wide` is fine here